common_features.mk

```diff
@@ -82,19 +82,19 @@ endif
 
 ifeq ($(strip $(UCIS_ENABLE)), yes)
     OPT_DEFS += -DUCIS_ENABLE
-    UNICODE_COMMON = yes
+    UNICODE_COMMON := yes
```

Comment:
Why all the changes here?

Reply:
because you requested `RAW_ENABLE := yes`, just being consistent